src/transformStateWithClones.js

```diff
@@ -7,7 +7,39 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const res = [];
+  let newState = {
+    ...state,
+  };
```

Comment:
Good naming should be in all parts of the code
<img width="841" alt="image" src="https://github.com/mate-academy/js_stateful-clones/assets/54860641/adf626e1-e310-4a6a-b099-d4c1e5034e17">
```suggestion
  const transformedStates = [];
  const stateCopy = { ...state };
```